### PR TITLE
renames CP to a less egregious name, Combat Strike

### PR DESCRIPTION
**Pull request by chizzy376**
## About The Pull Request
I honestly don't know if this is what will fix it I just dislike the obvious joke of Child #### being made when people vote CP
## Why It's Good For The Game
Less FBI visits for alleged CP
## Changelog
:cl:
fix: Combat Patrol no longer makes us look like CM staff team
/:cl:

### CODE REVIEW

File: code/datums/gamemodes/combat_patrol.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /datum/game_mode/combat_patrol
-	name = "Combat Patrol"
+	name = "Merc War"
```
Comment (Pariah919) [inline code/datums/gamemodes/combat_patrol.dm:2]:
```suggestion
	name = "Combat Strike"
```

Comment (chizzy376) [reply code/datums/gamemodes/combat_patrol.dm:2]: Yeah sure that sounds better

**Review by ivanmixo — CHANGES_REQUESTED** [review]
You didn't even change the path or anything else.

**Comment by chizzy376** [comment]
I'm too high for this shit

**Pull request closed without merging**